packages/emitter-framework/src/python/name-policy.ts

```diff
@@ -0,0 +1,31 @@
+import { createNamePolicy, NamePolicy } from "@alloy-js/core";
+import { constantCase, pascalCase, snakeCase } from "change-case";
+
+export type PythonElements =
+  | "class"
+  | "classMember"
+  | "enum"
+  | "enumMember"
+  | "function"
+  | "parameter"
+  | "constant"
+  | "variable";
+
+export function createPythonNamePolicy(): NamePolicy<PythonElements> {
+  return createNamePolicy((name, element) => {
+    switch (element) {
+      case "class":
+      case "enum":
+        return pascalCase(name);
+      case "enumMember":
+      case "constant":
+        return constantCase(name);
+      default:
+        return snakeCase(name);
+    }
+  });
+}
+
+export function usePythonNamePolicy(): NamePolicy<PythonElements> {
```

Comment:
Should that not be cached/use some context?

Reply:
Like cache the names? I just followed what Java did here.

Reply:
This is not how it should work. This should be just a `useNamePolicy()` with a type cast. The name policy needs to come from context.

Reply:
Copy the TypeScript one. I'll look into getting the Java one fixed too.